HousingFinanceInterimApi.Tests/V1/Gateways/UPCashFileNameTests.cs

```diff
@@ -47,11 +47,10 @@ public async Task GetTest(int value)
             Assert.Equal(fileName, getResult.FileName);
         }
 
-        public async Task GetNotExists()
+        public Task GetNotExists()
         {
-
+            return Task.CompletedTask;
         }
-
     }
```

Comment:
What is this?

Reply:
Looks like running dotnet format or dotnet-format removed some line spaces and added that line for some reason. I could remove this test, as it was empty before?

Reply:
If that's a test, then it's not very useful - always passes. Can be removed.